src/Renderer/Plates/PlatesRenderer.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types=1);
+
+namespace UniGen\Renderer\Plates;
+
+use League\Plates\Engine;
+use UniGen\Config\Config;
+use UniGen\Config\Exception\ConfigException;
+use UniGen\Renderer\Context;
+use UniGen\Renderer\RendererInterface;
+use UniGen\Renderer\ScalarValueMapper;
+
+class PlatesRenderer implements RendererInterface
+{
+    /** @var Config */
+    private $config;
+
+    /** @var Engine */
+    private $engine;
+
+    /**
+     * @param Config $config
+     *
+     * @throws ConfigException
+     */
+    public function __construct(Config $config)
+    {
+        $this->config = $config;
+        $this->engine = $this->createEngine();;
+    }
+
+    /**
+     * {@inheritDoc}
```

Comment:
Please be consistend, you're using different inherit doc formats. In PSR-19 (still in draft) we should use {@inheritDoc}